arangod/Cluster/EnsureIndex.cpp

```diff
@@ -90,6 +90,12 @@ EnsureIndex::EnsureIndex(MaintenanceFeature& feature,
 
 EnsureIndex::~EnsureIndex() = default;
 
+// For local book keeping and reporting on /_admin/actions
+arangodb::Result EnsureIndex::setProgress(double d) {
+  _progress = static_cast<uint64_t>(d);
```

Comment:
Why is `_progress` an `uint64_t`, if the API seems to suggest that `double` is used to express a percentage? Please clarify.

Reply:
It seems it is in fact a `double`, so why casting it to `uint64_t` here?

Reply:
For this I have no words. Simple fix. Obviously oversight. Left over from the bad idea in the `Maintenance` to use `uint64_t` as progress. One line change.